runtime/jvmti/jvmtiForceEarlyReturn.c

```diff
@@ -163,7 +163,7 @@ jvmtiForceEarlyReturn(jvmtiEnv* env,
 
 			if ((currentThread != targetThread)
 #if JAVA_SPEC_VERSION >= 21
-			&& (0 == J9OBJECT_U32_LOAD(currentThread, threadObject, vm->isSuspendedInternalOffset))
+			&& (J9_ARE_NO_BITS_SET(J9OBJECT_U64_LOAD(currentThread, threadObject, vm->internalSuspendStateOffset), J9_VIRTUALTHREAD_INTERNAL_STATE_SUSPENDED))
```

Comment:
the purpose adding the helpers goes away if we are not consistently using them to ease readability and maintenance. it might be straightforward to convert these into C++ files. if not, a simple approach would be to add wrapper methods in `jvmtiHelpers.cpp` which can invoke the helper methods in `VM_VMHelpers::*`. All JVMTI code should the wrapper methods in `jvmtiHelpers.cpp`, which should be inlined by the compiler. in the long run, we can transform the `.c` files into `.cpp` as part of our C++ transformation effort.

Reply:
> i also noticed a few other locations where the new helper can be used.

this applies to jvmtiLocalVariable.c and jvmtiStackFrame.c

also, jvmtiForceEarlyReturn.cpp requires a new line at the end of the file.

Reply:
> I've converted the file into cpp, local build/testing on JDK 22 passed, will launch a personal build with 8/11 to verify this didn't break any other JDK level

the cpp conversion changes look safe. i will launch the builds in the PR to save time.